### ISSUE TITLE
How to saved best checkpoint

### ISSUE THREAD

**Issue by syedali1621** [keras-team/keras-io#521]
In image_classification_efficientnet_fine_tuning.py after training how to save the checkpoint to convert it to .h5 format ?

**Comment by SuryanarayanaY** [comment]
Hi @syedali1621 ,

Keras has option to save `ModelCheckpoint` using [tf.keras.callbacks.ModelCheckpoint](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/ModelCheckpoint) where this class has a parameter called `save_best_only` which can be passed `True` to save only best model parameters based on some metric/loss which can be passed to the argument `monitor`.  This API looks at the parameter value of `monitor` in each training step and stores the best Model parameters.

We need to pass this callback to the argument `callbacks`  in model.fit(..., callbacks=[model_checkpoint_callback]) to enable the model saving best model parameters.

You can find a demo of same [here](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/ModelCheckpoint#example_2).

Hope this helps.If you want more details let us know. Thanks!
